scripts/initialize-conductor-data.js

```diff
@@ -43,25 +42,26 @@ async function populateHoloFuelData () {
 
   // 4.) Scenario: A request from agent1 to agent2, which agent 2 has paid :
```

Comment:
This Scenario is giving some issues and is not consistent. 
Due to time restrictions, I have just commented it out.

Reply:
This scenario does not work because of some networking issues. The data is not propagated on time.